examples/framesjs-starter/app/examples/simplified-api/new-api.tsx

```diff
@@ -0,0 +1,286 @@
+import assert from "assert";
```

Comment:
This file contains framework agnostic frames building, you can use this one insteead of React. React is just a wrapper around it. So perhaps the nextjs route could be fixed by removing react API and use this one directly.

This API is not complete, I wanted to open the discussion.

Reply:
What is the reason we are sticking to the react-like syntax? Feels like it is forcing us to keep all the logic contained within a single component

Reply:
React just because whole frames were mainly promoted as nextjs thingy right? So I guess it is nice to also provide React syntax although in my opinion it unnecessarily limits us in order to provide the easiest API possible.

Also I think you meant to comment on react api right? Because this file doesn't contain React API, this is agnostic API using simple classes.

Reply:
Also another question, how would you like to built frames apps? I think it's easier to think about some non abstract examples.

Reply:
Ok so essentially just keep the API as is but change how `post` button works to use paths so you can have multiple handlers.

I still think that having some sort of router is closer to react developers than having multiple route files. But maybe that could be abstracted afterwards.

Reply:
I would rename the `getFrameMessage` as it returns message and state.